examples/can/python-can/kvaser/receive_message_asyncio.py

```
@@ -1,49 +1,42 @@
+"""Receive asynchronously a message using Diagnostic on CAN protocol (ISO 15765)."""
+
 import asyncio
 
-from can import Bus, Message
-from uds.can import CanAddressingFormat, CanAddressingInformation
-from uds.transport_interface import PyCanTransportInterface
+from can import Bus
+from uds.can import CanAddressingFormat, CanAddressingInformation, PyCanTransportInterface
 
 
 async def main():
```

Ruff (0.12.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)